javagen/src/main/java/com/azure/autorest/util/MethodUtil.java

```diff
@@ -55,4 +57,18 @@ public static boolean isMethodIncludeRepeatableRequestHeaders(ProxyMethod proxyM
     public static boolean isHttpMethodSupportRepeatableRequestHeaders(HttpMethod httpMethod) {
         return REPEATABILITY_REQUEST_HTTP_METHODS.contains(httpMethod);
     }
+
+    public static Request findDPGRequestTobeProcessed(List<Request> requests) {
+        // if there is request with binary type, find the request consumes binary type
+        // if all requests are non-binary type, get the first request
```

Comment:
Just `findFirstBinaryRequest`?

Also move the comment to Javadoc of the method.